com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs

```diff
@@ -457,36 +457,53 @@ protected void PropertyField(SerializedDataParameter property)
         }
 
         /// <summary>
-        /// Draws a given <see cref="SerializedDataParameter"/> in the editor using a custom label
-        /// and tooltip.
+        /// Handles unity built-in decorators (Space, Header, Tooltips, ...) from <see cref="SerializedDataParameter"/> attributes
         /// </summary>
-        /// <param name="property">The property to draw in the editor.</param>
-        /// <param name="title">A custom label and/or tooltip.</param>
-        protected void PropertyField(SerializedDataParameter property, GUIContent title)
+        /// <param name="property">The property to obtain the attributes and handle the decorators</param>
+        /// <param name="title">A custom label and/or tooltip that might be updated by <see cref="TooltipAttribute"/> and/or by <see cref="InspectorNameAttribute"/></param>
+        void HandleDecorators(SerializedDataParameter property, GUIContent title)
         {
-            // Handle unity built-in decorators (Space, Header, Tooltip etc)
+            if (!property.attributes.Any())
+                return;
+
             foreach (var attr in property.attributes)
```

Comment:
Not sure you gain anything here. If there is any attribute, you will exit quickly of the foreach. So there is no gain to check if this is empty right above.

Also, if the attribute property do some operation, now it is done two times instead of just once. As this is reflection API I would be extra careful with this.

Reply:
Removed, thanks :)